code/modules/chemistry/tools/syringes.dm

```diff
@@ -286,10 +295,29 @@
 	name = "syringe (synaptizine)"
 	desc = "Contains synaptizine, a mild stimulant to increase alertness."
 	initial_reagents = "synaptizine"
-	
+
 /obj/item/reagent_containers/syringe/formaldehyde
 	name = "syringe (embalming fluid)"
 	desc = "Contains formaldehyde, a chemical that prevents corpses from decaying."
 	initial_reagents = "formaldehyde"
+
+/obj/item/reagent_containers/syringe/baster
+	name = "baster"
+	desc = "For adding delicious liquids to food."
+	icon_prefix = "baster"
+	icon_state = "baster_0"
+	initial_volume = 100
+	amount_per_transfer_from_this = 25
+
+	afterattack(var/atom/target, mob/user, flag)
+		switch (mode)
+			if (S_DRAW)
+				if (!istype(target, /obj/item/reagent_containers))
+					return
+			if (S_INJECT)
```

Comment:
Would like some kind of user feedback here- something like "You can't fit the nozzle in that!" so people don't get confused when nothing is happening